opennlp-tools/src/main/java/opennlp/tools/ml/AbstractEventTrainer.java

```diff
@@ -63,12 +63,10 @@ public boolean isValid() {
 
   public DataIndexer getDataIndexer(ObjectStream<Event> events) throws IOException {
 
-    String dataIndexerName = parameters.getStringParam(DATA_INDEXER_PARAM,
+    String dataIndexerName = trainingParameters.getStringParameter(DATA_INDEXER_PARAM,
         DATA_INDEXER_TWO_PASS_VALUE);
-    Map<String,String> indexParams = new HashMap<String, String>();
-    indexParams.put(AbstractDataIndexer.CUTOFF_PARAM, Integer.toString(getCutoff()));
```

Comment:
No, Cutoff is assigned in the TrainingParameters that come into AbstractEventTrainer.  I am concerned that the trainer/indexer might have different values of the cutoff.  I believe the default is "5".  I will double check where the default is set.